code/game/machinery/syndicatebomb.dm

```diff
@@ -292,17 +298,22 @@
 	beepsound = 'sound/items/bikehorn.ogg'
 
 /obj/machinery/syndicatebomb/empty
-	name = "bomb"
+	name = "bomb casing"
 	icon_state = "base-bomb"
-	desc = "An ominous looking device designed to detonate an explosive payload. Can be bolted down using a wrench."
+	desc = "A plasteel bomb casing. Load a payload and mend the wires before use. Can be bolted down using a wrench."
 	payload = null
 	open_panel = TRUE
-	timer_set = 120
 
 /obj/machinery/syndicatebomb/empty/New()
 	..()
 	wires.cut_all()
 
+/obj/machinery/syndicatebomb/empty/activate()
+	name = "bomb"
+	desc = "A plasteel bomb. Can be bolted down using a wrench."
+	..()
```

Comment:
Is this meant to be in `activate()`? seems off.

Reply:
It is, it's to rename the bomb once activated so it doesnt show as 'bomb casing' anymore.  If you can think of a better way to do that, I can change it.

Reply:
I feel like theres a better proc for this, such as when the core is inserted. Please have a look around to see if such proc exists.

Reply:
I'll take a look when I got the time and update this.